### ISSUE TITLE
[Effect Suggestion] Summon Bigger Fish

### ISSUE THREAD

**Issue by SeanMirrsen** [gta-chaos-mod/ChaosModV#455]
Summon Bigger Fish is about the most succinct I can imagine it being named. Spawn Obstructively Large Object is a bit too long, and possibly misleading since it works differently from other "spawn" effects.

The effect would pick one item from a list of Obstructively Large Objects, and spawn it at the point the player is currently looking at, or some distance away in that direction if the point is too far or player looks into the sky. Like it "cast a spell" in that direction. Obviously inspired by the 'I cast Summon Bigger Fish' gag from Darths&Droids.

The list of Obstructively Large Objects would include but not be limited to: a whale, an orca, a Tug, a Bus, a Rhino, a Dump truck, a cargo plane, a 'learjet' plane, a huge boulder from Meteor Rain, the Ferris Wheel, a cargo container, a cargo container handler, a bulldozer, a drill vehicle from that heist mission, any one of the semitrailers, and anything else I can't think of at the moment. Could be weighted towards the actual fish/marine animals, to make the effect's name literal more often.

Key difference from other "spawn something" effects - this spawns the object in the direction the player is going or looking. Since most of the time a player will be driving or running somewhere, the object will serve to obstruct, rather than crush, the player. It probably won't make much difference to the chance of softlocking, since spawned objects persist anyway.

**Comment by anto4440** [comment]
Either it would instantly kill the player or have no effect whatsoever?

**Comment by SeanMirrsen** [comment]
No, that's how the current "spawn thing" effects work. Like the tug, or the ferris wheel. Either it immediately kills the player, or softlocks them with them being unable to progress, or does nothing. This effect would spawn the object in the direction the player is looking at, meaning it would be unlikely to kill the player (unless they're in a very small space or looking at the floor), have a similar chance of softlocking, or could do nothing - but could also create an obstruction for the player to overcome. Overall more interesting than just variations on dropping something big on the player's head.

**Comment by anto4440** [comment]
Ok, makes more sense
